lib/datasource/remote/firebase/firebase_remote_config.dart

```diff
@@ -1,102 +1,71 @@
 
   void refresh() {
-    // Config has not been init yet. Dont call this function.
-    // if (_remoteConfig == null) {
-    //   return;
-    // }
-
-    // _remoteConfig.fetch().then((value) {
-    //   print(" _remoteConfig fetch worked");
-    //   _remoteConfig.activate().then((bool value) {
-    //     print(" _remoteConfig activate worked params were activated " + value.toString());
-    //   }).onError((error, stackTrace) {
-    //     print(" _remoteConfig activate failed");
-    //   });
-    // }).onError((error, stackTrace) {
-    //   print(" _remoteConfig fetch failed");
-    // });
+    _remoteConfig.fetch().then((value) {
+      print(" _remoteConfig fetch worked");
+      _remoteConfig.activate().then((bool value) {
+        print(" _remoteConfig activate worked params were activated $value");
+      }).onError((error, stackTrace) {
+        print(" _remoteConfig activate failed");
+      });
+    }).onError((error, stackTrace) {
+      print(" _remoteConfig fetch failed");
+    });
   }
 
   Future initialise() async {
-    // _remoteConfig = await RemoteConfig.instance;
-    //
-    // /// Maximum age of a cached config before it is considered stale. we set to 60 secs since we store important data.
-    // await _remoteConfig.setConfigSettings(RemoteConfigSettings(
-    //   minimumFetchInterval: const Duration(seconds: 60),
-    //   fetchTimeout: const Duration(seconds: 60),
-    // ));
-    //
-    // await _remoteConfig.setDefaults(defaults);
-    // refresh();
-  }
-
-  bool get featureFlagGuardiansEnabled => false; //_remoteConfig.getBool(_featureFlagGuardianKey);
-
-  String get termsAndConditions => _termsAndConditionsDefaultUrl; //_remoteConfig.getString(_termsAndConditionsUrlKey);
+    _remoteConfig = RemoteConfig.instance;
+    await _remoteConfig.setDefaults(defaults);
 
-  String get privacyPolicy => _privacyPolicyUrl; //_remoteConfig.getString(_privacyPolicyKey);
+    /// Maximum age of a cached config before it is considered stale. we set to 60 secs since we store important data.
+    await _remoteConfig.setConfigSettings(RemoteConfigSettings(
+      minimumFetchInterval: const Duration(seconds: 60),
+      fetchTimeout: const Duration(seconds: 60),
+    ));
 
-  String get hyphaEndPoint => _hyphaEndPointUrl; //_remoteConfig.getString(_hyphaEndPointUrl);
-
-  String get explorerUrl => _explorerUrl; //_remoteConfig.getString(_explorerUrlKey);
-
-  String get dhoExplorerUrl => _explorerUrl; //_dhoExplorerUrl;//_remoteConfig.getString(_dhoExplorerUrlKey);
+    refresh();
+  }
 
-  String get defaultEndPointUrl => _defaultEndPointUrl; //_remoteConfig.getString(_defaultEndPointUrlKey);
+  String get hyphaEndPoint => _remoteConfig.getString(_hyphaEndPointUrl);
 
-  String get defaultV2EndPointUrl => _defaultV2EndpointUrl; //_remoteConfig.getString(_defaultEndPointUrlKey);
+  String get defaultEndPointUrl => _remoteConfig.getString(_defaultEndPointUrlKey);
```

Comment:
v2 needs to be kept separate from non-v2, these need to be 2 different endpoints

Reply:
done